deploy/config/k8s/test-env-push-to-file.sh.yml

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+set -euo pipefail
+
+# Test cases can modify following variables from outside. Either by selecting
```

Comment:
I think we can leave out this comment section. These comments would make sense if the script was using environment variable settings that are set to a space-separated key/value pair, such as the definitions here: 
https://github.com/cyberark/secrets-provider-for-k8s/blob/main/deploy/config/k8s/test-env.sh.yml#L9-L11